### PR TITLE
update riskyuser schema

### PR DESCRIPTION
**Pull request by cloudhandler**
removes risk complex type

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811070840169060-3350%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zjXtQUqg%2BOmZh5JUsVYqkVBNeLCozMUU4u2Xzcp9HG8%3D&st=2018-11-07T08%3A36%3A03Z&se=2018-12-08T08%3A41%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/riskyuser.md](https://github.com/cloudhandler/microsoft-graph-docs/blob/patch-3/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811082152076894-3350%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oxJZqkN3Dvtw99XagGzKRzFVqmGK1DBdo9iYY%2F6LkUw%3D&st=2018-11-08T21%3A48%3A49Z&se=2018-12-09T21%3A53%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by cloudhandler** [comment]
#sign-off

File: api-reference/beta/api/teamsappinstallation_list.md [hunk 1]
```diff
@@ -67,6 +79,70 @@ Content-length: 55
 }
 ```
 
+## Example -- getting the names of the installed apps
```
Comment (Lauragra) [inline api-reference/beta/api/teamsappinstallation_list.md]: Please change the previous h2 to "Examples" and make this heading an H3. This way the heading structure stays consistent with the API topics. You can add an H3 for the first example so each example is organized under a peer heading.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/riskyuser.md](https://github.com/cloudhandler/microsoft-graph-docs/blob/patch-3/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811131842223047-3350%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hv%2FkJX6i6F70H4xLNxwKbqW1zmbCoAPWPyeF8u2soUk%3D&st=2018-11-13T18%3A39%3A31Z&se=2018-12-14T18%3A44%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/riskyuser.md](https://github.com/cloudhandler/microsoft-graph-docs/blob/patch-3/api-reference/beta/resources/riskyuser.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811131932412253-3350%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Br3qsoMrQ3Px9%2FjRAxmNcK4oMtM7sxPCTtBrbrJMqrY%3D&st=2018-11-13T19%3A29%3A52Z&se=2018-12-14T19%3A34%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**